core/invoker/src/main/resources/application.conf

```diff
@@ -120,5 +120,14 @@ whisk {
 
   invoker {
     protocol: http
+
+    activations {
+      # Enable this if ActivationStore is to be disabled and all activation results are
+      # to be sent to Kafka topic for further processing by ActivationPersisterService
+      activation-store-enabled = true
```

Comment:
this is strangely named: you enable this property to send to kafka instead of the "activation store" (which i'd think is the existing store)...

Reply:
Description here was wrong. Instead it would be

```
# Disable this if ActivationStore should not be used and all activation results are
# to be sent to Kafka topic for further processing by ActivationPersisterService
```